### PR TITLE
Fix surrogates

### PR DESCRIPTION
**Pull request by KEggensperger**
Make each benchmark download data automatically, rm old benchmarks.

@aaronkl: Just in case you want to have a look at it. I'll probably merge tomorrow.

### CODE REVIEW

File: hpolib/benchmarks/surrogates/cnn.py [hunk 1]
```diff
@@ -48,9 +45,16 @@ def get_configuration_space():
...
-                           [0, 1],  # batch_size
-                           [0, 1],  # n_units_1
-                           [0, 1],  # n_units_2
-                           [0, 1]],  # n_units_3
+                'bounds': [[0, 1],  # init_learning_rate, [10**-6, 10**0]
+                           [0, 1],  # batch_size, [32, 512]
+                           [0, 1],  # n_units_1,  [2**4, 2**8]
+                           [0, 1],  # n_units_2,  [2**4, 2**8]
+                           [0, 1]],  # n_units_3, [2**4, 2**8]
+                'references': ["@article{klein-corr16,"
```
Comment (aaronkl) [inline hpolib/benchmarks/surrogates/cnn.py]: that is actually the wrong reference the correct one is:
@InProceedings{klein-iclr17,
 author    = {A. Klein and S. Falkner and J. T. Springenberg and F. Hutter},
 title     = {Learning Curve Prediction with {Bayesian} Neural Networks},
 booktitle = {International Conference on Learning Representations (ICLR) 2017 Conference Track},
 year      = {2017},
 month 	= apr
}

File: hpolib/benchmarks/surrogates/svm.py [hunk 2]
```diff
@@ -47,7 +46,7 @@ def get_configuration_space():
     @staticmethod
     def get_meta_information():
         return {'name': 'Support Vector Machine',
-                'bounds': [[-10, 10],  # C
+                'bounds': [[-10, 10],   # C
                            [-10, 10]],  # gamma
                 'references': ["@article{klein-corr16,"
```
Comment (aaronkl) [inline hpolib/benchmarks/surrogates/svm.py]: @INPROCEEDINGS{klein-ejs17,
 author    = {A. Klein and S. Falkner and S. Bartels and P. Hennig and F. Hutter},
 title     = {Fast Bayesian hyperparameter optimization on large datasets},
 booktitle = {Electronic Journal of Statistics},
 year      = {2017},
 volume    = {11}
}

File: hpolib/benchmarks/surrogates/cnn.py (hunk 1 above)
Comment (KEggensperger) [reply hpolib/benchmarks/surrogates/cnn.py]: Agreed and fixed

File: hpolib/benchmarks/surrogates/svm.py (hunk 2 above)
Comment (KEggensperger) [reply hpolib/benchmarks/surrogates/svm.py]: Agreed and done

**Review by aaronkl — APPROVED** [review]

**Pull request merged**